orderable/tests/test_manager.py

```diff
@@ -0,0 +1,66 @@
+from django.test import TestCase
+
+from .models import SubTask, Task
+
+
+class TestOrderableManager(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.tasks = []
+        for i in range(3):
+            task = Task(sort_order=i)
+            task.save()
+            cls.tasks.append(task)
+
+    def test_gets_next(self):
+        first_task = self.tasks[0]
+        next_task = Task.objects.after(first_task)
+        self.assertEqual(next_task, self.tasks[1])
```

Comment:
These tests might be a little easier to read without the indexing:

```python
result = Task.objects.after(self.first_task)
self.assertEqual(result, self.middle_task)
```

This would of course require a slightly less DRY `setUpTestData`:
```python
self.first_task = Task.objects.create(sort_order=0)
self.middle_task = Task.objects.create(sort_order=1)
self.last_task = Task.objects.create(sort_order=2)
```